build.rs

```diff
@@ -638,6 +640,14 @@ fn gn() -> String {
   env::var("GN").unwrap_or_else(|_| "gn".to_owned())
 }
 
+/*
+ * Get the system's python binary - specified via the PYTHON environment
+ * variable or defaulting to `python3`.
+ */
+fn python() -> String {
+  env::var("PYTHON").unwrap_or_else(|_| "python3".to_owned())
```

Comment:
Thanks, this allows me to remove a hack I was using:

```sh
# workaround to use python3
# where "python" is invoked
mkdir -p "$(pwd)/bin"
ln -sf %{_bindir}/python3 "$(pwd)/bin/python"
export PATH="$PATH:$(pwd)/bin"
```

Reply:
Happy to hear that!